src/models/ProposalMessage.ts

```diff
@@ -6,12 +6,28 @@ export type Instrument = {
   allocatedTime: number;
 };
 
+export enum ProposalStatusDefaultShortCodes {
+  DRAFT = 'DRAFT',
+  FEASIBILITY_REVIEW = 'FEASIBILITY_REVIEW',
+  NOT_FEASIBLE = 'NOT_FEASIBLE',
+  SEP_SELECTION = 'SEP_SELECTION',
+  SEP_REVIEW = 'SEP_REVIEW',
+  ALLOCATED = 'ALLOCATED',
+  NOT_ALLOCATED = 'NOT_ALLOCATED',
+  SCHEDULING = 'SCHEDULING',
+  EXPIRED = 'EXPIRED',
+  EDITABLE_SUBMITTED = 'EDITABLE_SUBMITTED',
+  EDITABLE_SUBMITTED_INTERNAL = 'EDITABLE_SUBMITTED_INTERNAL',
+}
+
 export type ProposalMessageData = {
   proposalPk: number;
   shortCode: string;
   title: string;
   abstract: string;
-  newStatus?: string;
+  callId: number;
```

Comment:
Could you update the **validateProposalMessage** function and test it based on this change?